fh-metrics/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fh-metrics",
   "description": "FeedHenry Metrics Server",
-  "version": "3.0.6-BUILD-NUMBER",
+  "version": "3.0.7-BUILD-NUMBER",
```

Comment:
On fh-messaging, I changed package.json and the code, so I bumped the minor. On fh-metrics, I just changed package.json, so I bumped the patch only.